example/interface.js

```diff
@@ -8,10 +8,19 @@
 //  See the accompanying file LICENSE or http://www.apache.org/licenses/LICENSE-2.0.html
 //
 
-import { Vircadia, DomainServer, AudioMixer, AvatarMixer, MessageMixer, Uuid } from "../dist/Vircadia.js";
+import {
+    Vircadia,
+    DefaultLogConfiguration,
```

Comment:
```
Uncaught SyntaxError: The requested module '../dist/Vircadia.js' does not provide an export named 'DefaultLogConfiguration'
```
An probably revert this whole change?

Reply:
I'm not getting this error either, I've added exports in the Vircadia.ts for this, not sure what might be wrong :/